### PR TITLE
Docker image documentation [14003]

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

File: docs/rst/installation/docker.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+############
+Docker image
+############
+
+A DDS Router Docker image is distributed with Ubuntu 20.04 base image.
+This image launches an instance of DDS Router that is configured using a *YAML* configuration file provided by the user
+and shared with the Docker container.
+The steps to run DDS Router in a Docker container are explained below.
+
+#.  Download the compressed Docker image in ``.tar`` format from the
```
Comment (jparisu) [inline docs/rst/installation/docker.rst:14]: Is there any way to add a line break between lines in this list? The page seems cramped.

**Review by jparisu — CHANGES_REQUESTED** [review]

File: docs/rst/installation/docker.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    .. code-block:: bash
+
+        load ubuntu-ddsrouter:<version>.tar
+
+    where ``version`` is the downloaded version of DDS Router.
+
+#.  Build a DDS Router configuration YAML file on the local machine.
+    This will be the DDS Router configuration file that runs inside the Docker container.
+    As an initial test you can use one of the configuration files provided in the
```
[the remark this answers is not in the document]
Comment (rsanchez15) [reply docs/rst/installation/docker.rst:28]: IDK either so I reformulated the sentence

File: docs/rst/installation/docker.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    into the ``/<dds_router_ws>/DDS_ROUTER_CONFIGURATION.yaml`` file, where ``dds_router_ws`` is the path of the
+    configuration file.
+    To make this accessible from the Docker container we will create a shared volume containing just
+    this file. This is explained in next point.
+
+#.  Run the Docker container executing the following command:
+
+    .. code-block:: bash
+
+        docker run -it \
```
[the remark this answers is not in the document]
Comment (rsanchez15) [reply docs/rst/installation/docker.rst:40]: No it doesn't. Thats why the target shared volumen (file in this case) is fixed.

**Review by jparisu — APPROVED** [review]
LGTM

Waiting CI

**Pull request merged**